lib/improver/cli/spot_extract.py

```diff
@@ -213,9 +212,14 @@ def main(argv=None):
     # Check whether a lapse rate cube has been provided and we are dealing with
     # temperature data.
     if (args.temperature_lapse_rate_filepath and
-            diagnostic_cube.name() == "air_temperature"):
+            result.name() == "air_temperature"):
 
         lapse_rate_cube = load_cube(args.temperature_lapse_rate_filepath)
+        if not lapse_rate_cube.name() == 'air_temperature_lapse_rate':
+            msg = ("A cube has been provided as a lapserate cube but does "
```

Comment:
I think `lapserate` is a typo - should really be `lapse rate`.

Reply:
Fixed.